pom.xml

```diff
@@ -520,6 +520,30 @@
                     </ignoredResourcePatterns>
                 </configuration>
             </plugin>
+             <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M2</version>
+        <executions>
+          <execution>
+            <id>enforce-maven</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireMavenVersion>
+                  <version>3.5.4</version>
```

Comment:
@gitcliff , you wan to require maven version 3.5.4 but  i guess Travis runs wilth  3.5.2 ,so definately this will fail on Travis